docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,471 @@
+if not any(s.name == "daily-data-processing" for s in schedules):
+    print("Schedule deleted successfully from ZenML!")
+else:
+    print("Schedule still exists in ZenML!")
+```
+
+### Step 6.2: Delete the Schedule from the Orchestrator (Required)
+
+{% hint style="warning" %}
+**CRITICAL**: Deleting a schedule from ZenML does NOT delete it from the orchestrator. If you only perform Step 6.1, your pipeline will continue to run on schedule in the orchestrator!
+{% endhint %}
+
+Here's how to delete the schedule from Vertex AI:
+
+```python
+from google.cloud import aiplatform
+
+# List all Vertex schedules matching our schedule name
+vertex_schedules = aiplatform.PipelineJobSchedule.list(
+    filter='display_name="daily-data-processing"',
+    location="us-central1" # insert your location here
+)
+
+# Delete matching schedules
+for schedule in vertex_schedules:
+    print(f"Deleting Vertex schedule: {schedule.display_name}")
+    schedule.delete()
+    
+# Verify deletion from Vertex
+remaining_schedules = aiplatform.PipelineJobSchedule.list(
+    filter='display_name="daily-data-processing"',
+    location="us-central1"
+)
+if not list(remaining_schedules):
+    print("Schedule successfully deleted from Vertex AI!")
+else:
+    print("Warning: Schedule still exists in Vertex AI!")
+```
+
+The procedure for deleting schedules varies by orchestrator. Always check your orchestrator's documentation for the correct deletion method.
+
+## Troubleshooting: Quick Fixes for Common Issues
+
+Here are some practical fixes for issues you might encounter with your scheduled pipelines:
+
+### Issue: Timezone Confusion with Scheduled Runs
+
+A common issue with scheduled pipelines is timezone confusion. Here's how ZenML handles timezone information:
+
+1. **If you provide a timezone-aware datetime**, ZenML will use it as is
+2. **If you provide a datetime without timezone information**, ZenML assumes it's in your local timezone and converts it to UTC for storage and communication with orchestrators
+
+For cloud orchestrators like Vertex AI, Kubeflow, and Airflow, schedules typically run in the orchestrator's timezone, which is usually UTC. This can lead to confusion if you expect a schedule to run at 9 AM in your local timezone but it runs at 9 AM UTC instead.
+
+To ensure your schedule runs at the expected time:
+
+```python
+from datetime import datetime, timezone
+import pytz
+from zenml.config.schedule import Schedule
+
+# Option 1: Explicitly use your local timezone (recommended)
+local_tz = pytz.timezone('America/Los_Angeles')  # Replace with your timezone
+local_time = local_tz.localize(datetime(2025, 1, 1, 9, 0))  # 9 AM in your timezone
+schedule = Schedule(
+    name="local-time-schedule",
+    cron_expression="0 9 * * *",
+    start_time=local_time  # ZenML will convert to UTC internally
+)
+
+# Option 2: Use UTC explicitly for clarity
+utc_time = datetime(2025, 1, 1, 17, 0, tzinfo=timezone.utc)  # 5 PM UTC = 9 AM PST
+schedule = Schedule(
```

Comment:
out of curiousity, did you verify that this translates perfectly down to the GCP at least? Im not sure :D

Reply:
Tested this. It does what the docs say it does.